<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkingHoursSettings</class>
 <widget class="QDialog" name="WorkingHoursSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QTimeEdit" name="time_end_intermedie"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_start_intermedie">
       <property name="text">
        <string>Start intermedie</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_end">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_end_intermedie">
       <property name="text">
        <string>End intermedie</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTimeEdit" name="time_end"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_start">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeEdit" name="time_start_intermedie"/>
     </item>
     <item row="1" column="1">
      <widget class="QTimeEdit" name="time_start"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="schedule"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_schedule">
       <property name="text">
        <string>Schedule</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WorkingHoursSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WorkingHoursSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
